### PR TITLE
DAT-4075, add extra tracking for CONTROL group

### PR DESCRIPTION
**Pull request by RafLeszczynski**
## Links

* https://wikia-inc.atlassian.net/browse/DAT-4075

## Description
Add CONTROL group to nav AB test experiment A (fab icons

## Reviewers
@SebastianMarzjan @idradm

### CODE REVIEW

File: front/main/app/components/application-wrapper.js [hunk 1]
```diff
@@ -185,11 +186,20 @@ export default Ember.Component.extend({
...
-	}),
+			return currentGroup &&
+				(
+					currentGroup !== this.get('navABTestDefaultGroup') ||
+					currentGroup !== this.get('navABTestControlGroup')
+				);
+		}
+	),
+
+	navABTestIsControlGroup: Ember.computed.match('navABTestCurrentGroup', this.get('navABTestDefaultGroup')),
```
Comment (idradm) [inline front/main/app/components/application-wrapper.js]: ``navABTestDefaultGroup`` shouldn't it be ``navABTestControlGroup``?

File: front/main/app/templates/components/application-wrapper.hbs [hunk 2]
```diff
@@ -29,6 +29,8 @@
 		navABTestChangeUI=navABTestChangeUI
 		navABTestIsFabSearchIcon=navABTestIsFabSearchIcon
 		siteHeadIconClick=(action 'siteHeadIconClick')
+		navABTestIsControlGroup=navABTestIsControlGroup
+		navABTestExperimentName=navABTestExperimentName
```
Comment (idradm) [inline front/main/app/templates/components/application-wrapper.hbs]: Why do you pass it?

File: front/main/app/components/site-head.js [hunk 3]
```diff
@@ -30,6 +30,14 @@ export default Ember.Component.extend(
 					this.trackClick('recent-wiki-activity-blue-dot', 'open-navigation');
 				}
 
+				if (this.get('navABTestIsControlGroup')) {
+					trackExperiment(this.get('navABTestExperimentName'), {
+						action: trackActions.click,
+						category: 'entrypoint',
+						label: 'site-head-icon'
```
Comment (idradm) [inline front/main/app/components/site-head.js]: Is this label a valid description?

File: front/main/app/templates/components/application-wrapper.hbs (hunk 2 above)
Comment (RafLeszczynski) [inline front/main/app/templates/components/application-wrapper.hbs]: used for tracking

File: front/main/app/components/site-head.js (hunk 3 above)
Comment (RafLeszczynski) [inline front/main/app/components/site-head.js]: consistent with other tests

File: front/main/app/components/application-wrapper.js (hunk 1 above)
Comment (RafLeszczynski) [inline front/main/app/components/application-wrapper.js]: :+1:

**Comment by SebastianMarzjan** [comment]
Let it ❄️

**Pull request merged**